middleware/src/main/java/br/ufrn/dimap/middleware/lifecycle/PoolingManager.java

```diff
@@ -0,0 +1,115 @@
+package br.ufrn.dimap.middleware.lifecycle;
+
+import java.util.LinkedList;
+import java.util.Queue;
+
+/**
+ * Implements the pooling manager to provide servants to the Lifecycle Manager.
+ * 
+ * @author André Winston
+ * @author Carlos Antônio
+ * @author Gabriel Victor
+ * @author Pedro Henrique
+ * @author Wilson Farias
+ * @version 1.1
+ *
+ * @param <T>
+ */
+public class PoolingManager<T> implements PoolingI<T>{
+	
+    private Queue<T> pool = new LinkedList<>();
```

Comment:
Consider to use ConcurrentLinkedQueue or LinkedBlockingQueue to avoid concurrency issues.

Reply:
Add final constraint to all attributes initialized only once.